specification/synapse/data-plane/Microsoft.Synapse/stable/2021-04-01/symsAPI.json

```diff
@@ -0,0 +1,2206 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ArtifactsClient",
+    "description": "SyMS API Service Client.",
+    "version": "2021-04-01"
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{endpoint}",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "$ref": "#/parameters/Endpoint"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/databases/ExecuteChange": {
```

Comment:
https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#performing-an-action

Reply:
> https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#performing-an-action

Understood, the customers have been using this API in it's current form and any change will break the existing contract.